pkg/internal/tbtctest/tbtctest.go

```diff
@@ -0,0 +1,132 @@
+// Package tbtctest contains scenarios meant to be used for Bitcoin-related
+// tests in the pkg/tbtc package. Here are the details of specific scenarios:
+//
+// - deposit_sweep_scenario_0.json: Bitcoin deposit sweep transaction in which
+//   three inputs (a P2WPKH main UTXO, a P2SH deposit, and a P2WSH deposit)
+//   were swept into a P2WPKH main UTXO. For reference see:
+//   https://live.blockcypher.com/btc-testnet/tx/435d4aff6d4bc34134877bd3213c17970142fdd04d4113d534120033b9eecb2e
+//
+// - deposit_sweep_scenario_1.json: Bitcoin deposit sweep transaction in which
+//   two inputs (a P2SH deposit, and a P2WSH deposit) were swept into a P2WPKH
+//   main UTXO. For reference see:
+//   https://live.blockcypher.com/btc-testnet/tx/f8eaf242a55ea15e602f9f990e33f67f99dfbe25d1802bbde63cc1caabf99668
+//
+// - deposit_sweep_scenario_2.json: Bitcoin deposit sweep transaction in which
+//   one input (a P2SH deposit) was swept into a P2WPKH main UTXO.
+//   For reference see:
+//   https://live.blockcypher.com/btc-testnet/tx/f5b9ad4e8cd5317925319ebc64dc923092bef3b56429c6b1bc2261bbdc73f351
+//
+// - deposit_sweep_scenario_3.json: Bitcoin deposit sweep transaction in which
+//   one input (a P2WSH deposit) was swept into a P2WPKH main UTXO.
+//   For reference see:
+//   https://live.blockcypher.com/btc-testnet/tx/9efc9d555233e12e06378a35a7b988d54f7043b5c3156adc79c7af0a0fd6f1a0
```

Comment:
If we decide to add the possibility to choose the deposit sweep transaction output type ([see comment](https://github.com/keep-network/keep-core/pull/3421#discussion_r1042249662)), we could add another scenario which results in a non-witness main UTXO output. We tested that in our [typescript implementation](https://github.com/keep-network/tbtc-v2/blob/8b9d2629bf4333e650a54f32a4da7cf86bf6785e/typescript/test/deposit-sweep.test.ts#L287).

Reply:
See https://github.com/keep-network/keep-core/pull/3421#discussion_r1044538034